x/configuration/handler.go

```diff
@@ -30,11 +30,13 @@ func handleUpdateFees(ctx sdk.Context, msg types.MsgUpdateFees, k Keeper) (*sdk.
 	return &sdk.Result{}, nil
 }
 
-func handleUpdateConfig(ctx sdk.Context, msg types.MsgUpdateConfig, k Keeper) (*sdk.Result, error) {
+func handleUpdateConfig(ctx sdk.Context, msg *types.MsgUpdateConfig, k Keeper) (*sdk.Result, error) {
 	configurer := k.GetConfigurer(ctx)
 	if !configurer.Equals(msg.Signer) {
 		return nil, sdkerrors.Wrapf(sdkerrors.ErrUnauthorized, "%s is not allowed to update configuration", msg.Signer)
 	}
+	msg.NewConfiguration.AccountRenewalCountMax = msg.NewConfiguration.AccountRenewalCountMax + 1
```

Comment:
Please add a note about why the renewal maxes are being increased by 1.

Reply:
Actually, I'd much rather have the +1 in https://github.com/iov-one/iovns/blob/master/x/domain/controllers/account/account.go#L173 and the `msg` not manipulated.  What's the argument against that @orkunkl / @fdymylja ?

Reply:
I would put the focus on the account controller too rather than the configuration itself, also because in this way we get inconsistency between input and output. 

Ex: I input 1 I query and get 2.

Although the most proper way would be to just accept the fact that creation accounts as 1 renewal, and so while updating configuration etc to take this into account. Orkun here is like "saying" to us that the code will understand this when in reality should be the configuration setter to understand that creation counts as 1 renewal.

Reply:
You are right I did not think of query case @fdymylja